CHANGELOG.md

```diff
@@ -225,6 +225,10 @@ Syntax
   `syntax f {x} = …`. However, `{x}` is interpreted as a pun in
   `λ (c {x}) → …`.
 
+* Records can now be created using module-like syntax in place of curly braces
```

Comment:
Maybe you could add a minimal example here?

Reply:
Done.